app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is defined as a class attribute. This means it is shared across all instances of the `Person` class. If this is intentional, ensure that you handle it carefully to avoid unexpected behavior.